packages/babel-plugin-import-jsx-pragma/README.md

```diff
@@ -60,6 +62,13 @@ _Type:_ String
 
 Name of variable required to be in scope for use by the JSX pragma. For the default pragma of React.createElement, the React variable must be within scope.
 
+### `scopeVariableFrag`
```

Comment:
Can we not abbreviate this? In general, I'd discourage abbreviations, as they tend to lose clarity for little gain.

Reply:
Indeed, as you noted (https://github.com/WordPress/gutenberg/pull/15120/files#r279532762) this was named for consistency with the [`@babel/plugin-transform-react-jsx` `pragmaFrag` option](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#pragmafrag). I think we're agreed that consistency is preferable here?

Reply:
> I think we're agreed that consistency is preferable here?

👍